app/fetch_preconfig.js

```diff
@@ -37,12 +37,14 @@ const handleRedirectProtocol = (url, response) => {
 };
 
 Client4.doFetchWithResponse = async (url, options) => {
-    if (!Client4.online) {
-        throw new ClientError(Client4.getUrl(), {
-            message: 'no internet connection',
-            url,
-        });
-    }
+    // Removing the check of this flag to be handled natively.
```

Comment:
I think we should just remove this instead of commenting it out since it'll likely become out of date quickly.

Also, if we're removing this, should we remove it from the redux version of this method as well?

Reply:
I agree on removing it but it's Elias' idea to just comment it out and I do get it as experimental. I could create a ticket for next release to address cleaning it up including the removal of `Client4.online` on redux if it's fine with you.

Reply:
Yeah, a ticket would be nice so we can keep track of it for later. The code is fine for the release regardless

Reply:
Thanks, ticket filed - [MM-14619](https://mattermost.atlassian.net/browse/MM-14619)